### PR TITLE
refactor: #2098 clean up app detail redux tree

### PR DESCRIPTION
**Pull request by tanphamhaiduong**
# Pull request checklist

## Does this close any currently open issues?

fixes: #2098

**Please check if your PR fulfills the following requirements:**

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`yarn build`) was run locally and any changes were pushed
- [x] Lint (`yarn lint`) has passed locally and any fixes were made for failures
- [x] Test (`yarn test`) has passed locally and any fixes were made for failures

## Pull request type

**Please check the type of change your PR introduces:**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: #2098 

## What is the new behavior?

- Refactor

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/marketplace/src/sagas/installations/__tests__/app-installations.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ const uninstallParams = {
 
 const connectSession = 'connectSession'
 
-describe('app-installations sagas', () => {
+xdescribe('app-installations sagas', () => {
```
Comment (phmngocnghia) [inline packages/marketplace/src/sagas/installations/__tests__/app-installations.ts:33]: Is this typo?

Comment (phmngocnghia) [inline packages/marketplace/src/sagas/installations/__tests__/app-installations.ts:33]: should add TODO to fix test, and update to `describe` in next PR?

File: packages/marketplace/src/actions/negotiators/negotiators.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { actionCreator } from '@/utils/actions'
+import ActionTypes from '@/constants/action-types'
+import { FetchNegotiatorsParams } from '@/services/negotiators'
+import { PagedResultNegotiatorModel_ } from '@reapit/foundations-ts-definitions'
+
+export const fetchNegotiators = actionCreator<FetchNegotiatorsParams>(ActionTypes.FETCH_NEGOTIATORS)
+
+export const fetchNegotiatorsSuccess = actionCreator<PagedResultNegotiatorModel_>(ActionTypes.FETCH_NEGOTIATORS_SUCESS)
```
Comment (nphivu414) [inline packages/marketplace/src/actions/negotiators/negotiators.ts:8]: Should be `SUCCESS` not `SUCESS`

File: packages/marketplace/src/actions/negotiators/negotiators.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+export const fetchNegotiatorsFailure = actionCreator<string>(ActionTypes.FETCH_NEGOTIATORS_FAILURE)
```
Comment (nphivu414) [inline packages/marketplace/src/actions/negotiators/negotiators.ts:10]: Should be `FAILED` instead of `FAILURE` for consistency

Comment (tanphamhaiduong) [reply packages/marketplace/src/actions/negotiators/negotiators.ts:10]: I will fix in the next PR

File: packages/marketplace/src/actions/negotiators/negotiators.ts (hunk 2 above)
Comment (tanphamhaiduong) [reply packages/marketplace/src/actions/negotiators/negotiators.ts:8]: I will fix in the next PR

**Pull request merged**